config/magpie.ini

```diff
@@ -61,6 +61,10 @@ magpie.user_registration_notify_enabled = false
 magpie.user_registration_notify_email_recipient =
 magpie.user_registration_notify_email_template =
 
+# --- user validation settings ---
+
+#magpie.supplemental_username_regex =
```

Comment:
Should we keep the `supplemental` prefix since this regex is in addition to any existing built-in validate regexes.  It does not replaces any existing validation regexes.

Reply:
`magpie.user_name_extra_regex` is good to be more explicit
I think it could be relevant to add additional metadata in `ax.verify_param` that indicates that it is that regex, and not the default one that raises the error. Something like: `param_content={"setting": "magpie.user_name_extra_regex"}`.

Reply:
> `magpie.user_name_extra_regex` is good to be more explicit

Why is this name better than "magpie.supplemental_user_name_regex"?
- you prefer the word "extra" to "supplemental"?
- you want the setting to start with "user_name_"?
- something else?

I'm not opposed to changing it, I just don't understand what this change buys us.

Reply:
For reference: https://github.com/Ouranosinc/Magpie/pull/592#discussion_r1366176146